### PR TITLE
Adds factory methods to TagKey and TagValue

### PR DESCRIPTION
**Pull request by dinooliva**

### CODE REVIEW

File: core/java/com/google/instrumentation/stats/RpcConstants.java [hunk 1]
```diff
@@ -58,8 +60,13 @@
   public static final MeasurementDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY =
       MeasurementDescriptor.create(
           "/rpc/client/roundtrip_latency",
-          "RPC roundtrip latency us",
-          MeasurementUnit.create(-6, scalar));
+          "RPC roundtrip latency msecs",
+          MeasurementUnit.create(-3, scalar));
```
Comment (sebright) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: Why is this in milliseconds?

Comment (dinooliva) [inline core/java/com/google/instrumentation/stats/RpcConstants.java]: That shouldn't be part of this change - I've reverted it for now.
